### DISCUSSION TITLE
蓝灯充值2年，频繁连接不上，只刚开始使用三个小时，能否退款？

### DISCUSSION THREAD

**Discussion by zy0628zy** [getlantern/lantern-forum-cn#478] [Q&A]
蓝灯4月30日凌晨充值2年，我现在才使用三个小时就频繁掉线，能否退款？
![Screenshot_2024-04-30-00-45-29-91_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/3aa45f92-5c65-4e07-a899-4af5b1982302)
![Screenshot_2024-04-30-01-57-25-43_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/16f0b057-dc56-4ffc-b45b-66c2f1fb8092)
![Screenshot_2024-04-30-01-13-35-14_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/9a831e29-c32f-4259-a967-76e305f9fa3a)
这是我支付凭证
![Screenshot_2024-04-30-03-13-12-61_cf3cf72bd8e53b0db7ddb0a6f2208af9](https://github.com/getlantern/lantern-forum-cn/assets/168475629/afe5f9f3-4f6f-4ef6-8ad9-5cf7a5b537bc)

**Comment by zy0628zy** [comment]
![Screenshot_2024-04-30-03-38-38-97_6cf654f91021b6bc082d9f2a76a62578](https://github.com/getlantern/lantern-forum-cn/assets/168475629/572a9b2d-cdac-49d2-af80-33eee3e074a8)
邮件已发，是自动回复

**Comment by zy0628zy** [comment]
我的邮箱是zhangying06283027@126.com

**Comment by zy0628zy** [comment]
![Screenshot_2024-04-30-03-58-03-32_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/ec495d4b-3677-4130-8cf0-961f0c9fb7d3)

**Comment by zy0628zy** [comment]
![Screenshot_2024-04-30-09-14-13-09_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/9da43a99-10ff-4948-b675-ce1943806d0e)
![Screenshot_2024-04-30-09-06-12-23_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/632512e6-c184-41f0-ada2-c1fbf09a6657)
![Screenshot_2024-04-30-08-57-56-41_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/6986e7a7-77f5-4ac5-bcd8-939e131638d0)
永远上不上去

**Comment by MaxMKMax** [comment]
@zy0628zy 您好. 可以退款的. 但您要再试试看给我们机会吗? 今晚我会和技术团队开会来反馈您遇到的问题. 要不您再使用两个星期? 如果您真的需要退款, 我们也会帮您弄退款. 请您放心!


在这之前, 您可以帮助我们在应用程序中提交报告您的问题吗？您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by zy0628zy** [reply]
![Screenshot_2024-04-30-17-44-12-40_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/d138d8a7-fe61-4f81-9583-511d7ffaa65b)
![Screenshot_2024-04-30-17-43-38-32_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/2b3985a4-ca20-490e-9c0d-54c74710c2b4)
![Screenshot_2024-04-30-17-34-18-07_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/c5583b1a-72d6-4d66-b9b7-485d3a7f116a)
![Screenshot_2024-04-30-11-01-10-29_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/221a959d-23f0-4044-a70d-e53e82c9d388)
![Screenshot_2024-04-30-08-57-56-41_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/2452c14a-b633-48f5-9f9b-9f1da503f859)
白天已经尝试过多次，就根本上不上去了，无论美国还是新加坡的服务器都上不去。或者在线时间极短。不好意思，我申请退款。不愿意再尝试。thanks。

**Comment by zy0628zy** [comment]
![Screenshot_2024-05-02-00-19-39-00_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/17bc4629-f188-438c-bf38-eaae2afc88f6)
![Screenshot_2024-05-02-10-44-38-00_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/497a5f72-b676-4cd4-80ca-288d3f68f052)
![Screenshot_2024-05-02-10-54-05-87_4b7dd4d3471a9bd5811f20f1c9979b1f](https://github.com/getlantern/lantern-forum-cn/assets/168475629/60da1bda-89e0-4a82-9e3e-20a2ada088af)
